tests/peer_did_resolver_tests.py

```diff
@@ -0,0 +1,13 @@
+import unittest
+
+
```

Comment:
It would be great to create a demo test/script, showing how the API should be used.
It can be something like
```
peer_did_alg_0 = create_numalgo_0(...)
peer_did_alg_2 = create_numalgo_2(...)

save_peer_did_doc(peer_did_alg_0)
save_peer_did_doc(peer_did_alg_2)

did_doc_alg_0 = resolve_peer_did(peer_did_alg_0)
did_doc_alg_2 = resolve_peer_did(peer_did_alg_2)
```